src/AbstractFixer.php

```diff
@@ -45,6 +46,13 @@ public function __construct()
         }
     }
 
+    public function fix(\SplFileInfo $file, Tokens $tokens)
```

Comment:
No, this is not a BC breaker.
Side-projects shall use `FixerInterface`, which has only `fix` method (no `applyFix`).
`AbstractFixer` is marked as **internal** and must not be used outside of this package.